src/components/BuilderComponents/Iframe.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./styles/Iframe.scss";
+
+const Iframe = ({
+  bgColor,
+  content,
+  size: { height = 300, width = 300 },
+  title,
+}) => {
+  return (
+    <div className="iframe-container" style={{ backgroundColor: bgColor }}>
+      <iframe
+        title={title}
+        width={width}
+        height={height}
```

Comment:
hardcoding pixels is sure to cause pain when different screen sizes like mobile come into play

Instead, I'd recommend having the iframe fill the available space and have the "size limit" live inside a container

This seems to give a nice tutorial to get started with:
https://www.w3schools.com/howto/howto_css_responsive_iframes.asp